src/sentry/integrations/issues.py

```diff
@@ -15,6 +15,12 @@ class IssueBasicMixin(object):
     def get_group_title(self, group, event, **kwargs):
         return event.error()
 
+    def get_issue_url(self, key, **kwargs):
```

Comment:
can you add jira tickets to implement this for jira, bitbucket and vsts and assign them to me/lb accordingly?